mordant/src/commonMain/kotlin/com/github/ajalt/mordant/input/SelectListAnimation.kt

```diff
@@ -82,16 +82,36 @@ class InteractiveSelectListBuilder(private val terminal: Terminal) {
         description: String,
         selected: Boolean = false,
     ): InteractiveSelectListBuilder = apply {
-        config.entries += SelectList.Entry(title, description, selected)
+        config.entries += SelectList.Entry(title, description, selected, null)
+    }
+
+    /** Add an item to the list of items to select from */
+    fun addEntry(
```

Comment:
No need for the duplicated overloads here. We can get rid of the versions without the `value` parameter and keep the new versions as long we we annotate them with `@JvmOverloads` for backwards compatibility.